py/openage/pack/pack_config.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+import configparser
+import re
+
+
+class PackConfig:
+    """
+    main config file for a mod pack.
+
+    stores the package metainformation for the mod list.
+
+    file contents:
+
+    [openage-pack]
+    name = <package name>
+    version = <package version>
+    author = <pack author>
+    pack_type = (game|mod|...)
```

Comment:
Hm, yes, they are probably the same. I thought of this difference because then we could separate the download lists by mods and games. The engine can run Star Wars or AOE and maybe more, and these shouldn't be in the "mod" selection list i think.